integration_test/third_party_apps_test/main_test.go

```diff
@@ -821,7 +821,7 @@ type test struct {
 }
 
 var defaultPlatforms = map[string]bool{
-	"debian-cloud:debian-10":     true,
+	"debian-cloud:debian-12":     true,
```

Comment:
Should we not be testing on `debian-11` (the oldest available) by default? And then, why not `ubuntu-os-cloud:ubuntu-2004-lts` (which is actually older)?

Reply:
[9aa89a0](https://github.com/GoogleCloudPlatform/ops-agent/pull/1752/commits/9aa89a0523f0594583e8413e95c5dedaeac4f44d)